### PR TITLE
App and Row Quota Fixes

### PR DESCRIPTION
**Pull request by Rory-Powell**
## Description
- Migrate the app count to the correct number
- Remove quota monthly reset
- Disable row quota (and migrate back to 0)
- Disable other quotas (only apps and rows were working)

## Screenshots
Pre-migration
Screenshot 2022-01-11 at 18 43 52

Post-migration
Screenshot 2022-01-11 at 18 45 26

### CODE REVIEW

File: packages/server/src/migrations/sync_app_and_reset_rows_quotas.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+const { MIGRATIONS, MIGRATION_DBS, migrateIfRequired } =
+  require("@budibase/backend-core").migrations
```
Comment (mike12345567) [inline packages/server/src/migrations/sync_app_and_reset_rows_quotas.js:2]: Could we replace this with the new standard method `("@budibase/backend-core/migrations")`? I've put together a [PR](https://github.com/Budibase/budibase/pull/4004) to standardise it completely across the usages in the monorepo with some explanation as to the reasoning and the stuff I tested.

**Review by mike12345567 — APPROVED** [review]
LGTM - just one clean up, just updating a use of the backend core to be consistent with the latest PR.

Edit: noticed my refactor was conflicting, just jumped in to merge/refactor above issue!

**Review by shogunpurple — APPROVED** [review]
LGTM - we should create an issue to turn the `rows` one back on and include the fixes for table/app deletion, so there is some limit.

EDIT: https://github.com/Budibase/budibase/issues/4032

File: packages/server/src/middleware/usageQuota.js [hunk 2]
```diff
@@ -6,6 +6,9 @@ const {
   isExternalTable,
   isRowId: isExternalRowId,
 } = require("../integrations/utils")
+const quotaMigration = require("../migrations/sync_app_and_reset_rows_quotas")
+
+const testing = false
```
[the remark this answers is not in the document]
Comment (Rory-Powell) [reply packages/server/src/middleware/usageQuota.js:11]: @mike12345567 Updated in https://github.com/Budibase/budibase/pull/4087 to use env vars only for quotas, so can be enabled / disabled by adding `USE_QUOTAS` to `.env`

**Pull request merged**